frontend/src/components/newRequest/NewRequest.js

```diff
@@ -0,0 +1,212 @@
+  };
+
+  const handleDescriptionChange = (event) => {
+    setDescription(event.target.value);
+  };
+
+  const handleFirstLineChange = (event) => {
+    setFirstLine(event.target.value);
+  };
+
+  const handleCityChange = (event) => {
+    setCity(event.target.value);
+  };
+
+  const handlePostcodeChange = (event) => {
+    setPostcode(event.target.value);
+  };
+
+  const handleNeededByDateChange = (event) => {
+    setneededByDate(event.target.value);
+  };
+
+  return (
+    <main>
+      <h2 id="new-request-title">Add A Request</h2>
+      <div className="container">
+        <form className="newRequestForm" onSubmit={handleSubmit}>
+          <div className="input-box">
+            <label id="form_label" htmlFor="organisation-name">
+              Organisation Name
+            </label>
+            <input
+              className="form_field"
+              id="organisation-name"
+              type="text"
+              value={organisationName}
+              onChange={handleOrganisationNameChange}
+              placeholder="Cancer Research"
+            />
+            <i></i>
+          </div>
+
+          <div className="input-box">
+            <label id="form_label" htmlFor="title">
+              Title
+            </label>
+            <input
+              className="form_field"
+              id="title"
+              type="text"
+              value={title}
+              onChange={handleTitleChange}
+              placeholder="What do you need?"
+            />
+            <i></i>
+          </div>
+
+          <div className="input-box">
+            <label id="form_label" htmlFor="requirement">
+              Do you require volunteering or donation of goods?
+              <select value={requirement} onChange={handleRequirementChange}>
+              <option value="requirement">Volunteering</option>
+              <option value="resources">Donation of Goods</option>
+              </select>
+            </label>
+            <i></i>
+          </div>
+
+          <div className="input-box">
+            <label id="form_label" htmlFor="description">
+              Description
+            </label>
+            <input
+              className="form_field"
+              id="description"
+              type="text"
+              value={description}
+              onChange={handleDescriptionChange}
+              placeholder="E.g. 2 volunteers to help running of soup kitchen"
+            />
+            <i></i>
+          </div>
+
+          {/* -----------------address------------------- */}
+
+          <div className="input-box">
+            <label id="form_label" htmlFor="first-line">
+              First Line of Address
+            </label>
+            <input
+              className="form_field"
+              id="first-line"
+              type="text"
+              value={firstLine}
+              onChange={handleFirstLineChange}
+              placeholder="2 Redman Place"
```

Comment:
Should put 'e.g.' in front of the placeholders

Reply:
I put in placeholders for the time being but that is something we can discuss later on when we do styling as all the placeholders isn't appealing to the :face_exhaling: